pyfr/plugins/fluidforce.py

```diff
@@ -73,6 +88,19 @@ def __init__(self, intg, cfgsect, suffix):
                     m0[etype, fidx] = eles.basis.m0[facefpts]
                     qwts[etype, fidx] = eles.basis.fpts_wts[facefpts]
 
+                    if self.forcetype == 'viscous':
+                        m4e = eles.basis.m4.reshape(self.ndims, eles.nupts, -1)
+                        m4[etype, fidx] = np.einsum('ij,kjl->kil',
+                                                    m0[etype, fidx], m4e)
+
+                        # |J|^-1 Smat^T
+                        smat = eles.smat_at('upts').get().swapaxes(1, 2)
```

Comment:
Why do we not compute the smats at the flux points directly and then pick out the ones we want say by adding a `smat_at_np` function like we have for `ploc`?